web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -214,6 +240,7 @@ export const DailyViewTable = (props: Props): JSX.Element => {
           <Table stickyHeader aria-label="daily table view of HFI by planning area">
             <TableHead>
               <TableRow>
+                <TableCell></TableCell>
```

Comment:
Is the empty `TableCell` for spacing?

Reply:
yep. I should add a comment to that effect tho